[2022-07-28 10:13:23,764] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: user_processing.create_table manual__2022-07-28T10:13:22.952035+00:00 [queued]>
[2022-07-28 10:13:23,771] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: user_processing.create_table manual__2022-07-28T10:13:22.952035+00:00 [queued]>
[2022-07-28 10:13:23,772] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-28 10:13:23,773] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-07-28 10:13:23,773] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-28 10:13:23,787] {taskinstance.py:1377} INFO - Executing <Task(PostgresOperator): create_table> on 2022-07-28 10:13:22.952035+00:00
[2022-07-28 10:13:23,791] {standard_task_runner.py:52} INFO - Started process 900 to run task
[2022-07-28 10:13:23,794] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'manual__2022-07-28T10:13:22.952035+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpkzi4w4nq', '--error-file', '/tmp/tmpdzx1fqnq']
[2022-07-28 10:13:23,795] {standard_task_runner.py:80} INFO - Job 31: Subtask create_table
[2022-07-28 10:13:23,844] {task_command.py:369} INFO - Running <TaskInstance: user_processing.create_table manual__2022-07-28T10:13:22.952035+00:00 [running]> on host 890323d6f4a0
[2022-07-28 10:13:23,911] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2022-07-28T10:13:22.952035+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-07-28T10:13:22.952035+00:00
[2022-07-28 10:13:23,920] {base.py:68} INFO - Using connection ID 'postgres' for task execution.
[2022-07-28 10:13:23,923] {dbapi.py:208} INFO - Running statement: 
        CREATE TABLE IF NOT EXISTS users (

            firstname TEXT NOT NULL,
            lastname TEXT NOT NULL,
            country TEXT NOT NULL,
            username TEXT NOT NULL,
            password TEXT NOT NULL,
            email TEXT NOT NULL
        );
           , parameters: None
[2022-07-28 10:13:23,925] {postgres.py:94} INFO - NOTICE:  relation "users" already exists, skipping

[2022-07-28 10:13:23,936] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20220728T101322, start_date=20220728T101323, end_date=20220728T101323
[2022-07-28 10:13:23,966] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-28 10:13:24,012] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
